client/landing/jetpack-cloud/index.js

```diff
@@ -6,10 +6,22 @@ import page from 'page';
 /**
  * Internal dependencies
  */
-import { navigation, siteSelection } from 'my-sites/controller';
-import { makeLayout, render as clientRender } from 'controller';
-import { dashboard } from 'landing/jetpack-cloud/sections/dashboard/controller';
+import config from 'config';
 
 export default function() {
-	page( '/', siteSelection, navigation, dashboard, makeLayout, clientRender );
+	page( '/', context => {
+		let redirectPath = '/error';
```

Comment:
The default is `/error` which leads to a 404. Should we have a more graceful error here? 
we could redirect the user to jetpack.com. 

Since both of there things are a bit of a hypothetical error. I am fine if we merge this.

Reply:
yeah, My thinking was it looks like an error and that an error page should probably be a separate PR if we want something specific for Jetpack Cloud.